### PR TITLE
Rework and enhancement of Get Started guide

### PR DESCRIPTION
**Pull request by Beak-man**
This is the first draft for the new Get Started guide.

The idea is to leverage Web WorldWind as a web library in order to show live views of WebWW itself while going through the steps of different planned tutorials. This is the first test of that format.

The focus of this guide is to teach the newcomer to Web WorldWind some of its core concepts, while making distinctions useful for our seasoned Java and Android developers that are just beginning to test the web version and web development in general.

The guide cumulatively works through an example application, first showing how to display a basic globe with base imagery, deploying it in a test web server, and then displaying different features over the globe:

 - A placemark with its label
 - A surface Polygon
 - One COLLADA 3D model
 - A WMS-retrieved layer

The guide contains syntax-highlighted code examples to aid the new developer. They're extracts of what we currently have in WebWW's Examples, while further simplifying them (avoiding unnecessary configurations and in-depth coverage of more WorldWind concepts) an reordering them so they always follow the pattern of layer creation > shape attribute configuration > shape creation > shape addition to layer.

**Some potential concerns:**

- Out of "good practices" I'm proposing creating a Layer for each shape, but this is not strictly necessary. We could store all shapes in a single RenderableLayer in order to shorten the guide a bit. I chose not to do it in order to not produce confusion for the student in his later developments.

- When explaining some core concepts (_e.g._ "Layer") it may be considered useful to actually link the related API doc within the guide. I chose not to do so in order to decrease our maintenance requirements of a long list of hyperlinks, but this is up for consideration. That being said, I liberally linked to outside resources.

- In order to avoid confusion, I'm removing the mention to our npm package. I think that's a better fit for _e.g._ a tutorial that shows how to work with WorldWind within an Angular or React framework.

- The WMS section introduces a new library (jQuery) just before the guide is going to end, increasing its overall perceived complexity to someone unfamiliar with it. This may be considered for exclusion from this guide at least. I chose to keep it since retrieving imagery from an external web service is a core functionality of any mapping or 3D globe library, I think.

- The couple of bullet point lists in Markdown don't look too good. Couldn't find a way to make them look better, though.

### CODE REVIEW

File: content/web/get-started.md [hunk 1]
```diff
@@ -7,44 +7,451 @@ draft: false
...
+
+We should now visualize the average temperature imagery over land in the globe:
+
+{{% latestWorldWindScript url="https://files.worldwind.arc.nasa.gov/artifactory" repo="web" %}}
+{{% getStarted_WMS %}}
+
+---
+## In summary
+
+Congratulations! You now have a good grasp of many of the basic Web WorldWind concepts and features.
```
Comment (eirizarry) [inline content/web/get-started.md]: "Congratulations! You now have a good grasp of how to use core Web WorldWind concepts and features."

File: content/web/get-started.md [hunk 2]
```diff
@@ -7,44 +7,451 @@ draft: false
...
+
+Every feature here and more is described thoroughly via code examples in our repository in the ['examples'](https://github.com/NASAWorldWind/WebWorldWind/tree/develop/examples) and ['apps'](https://github.com/NASAWorldWind/WebWorldWind/tree/develop/apps) folders. Be sure to also check out our [API documentation](https://worldwind.arc.nasa.gov/web/docs/) for detailed information on the objects and functions described here.
+
+This is our final, resultant code:
```
Comment (eirizarry) [inline content/web/get-started.md]: Here is the final result in code:

File: content/web/get-started.md [hunk 3]
```diff
@@ -7,44 +7,451 @@ draft: false
...
 
-    <script src="{{% latestArtifactoryPath url="https://files.worldwind.arc.nasa.gov/artifactory" repo="web" %}}/worldwind.min.js"
-            type="text/javascript">
-    </script>
+wwd.addLayer(new WorldWind.CompassLayer());
+wwd.addLayer(new WorldWind.CoordinatesDisplayLayer(wwd));
+wwd.addLayer(new WorldWind.ViewControlsLayer(wwd));
+{{< /highlight >}}
 
 ---
```
Comment (eirizarry) [inline content/web/get-started.md]: Delete

File: layouts/shortcodes/getStarted_FinalResult.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        placemarkAttributes.labelAttributes.offset = new WorldWind.Offset(
+            WorldWind.OFFSET_FRACTION, 0.5,
+            WorldWind.OFFSET_FRACTION, 1.0);
+
+        placemarkAttributes.imageSource = WorldWind.configuration.baseUrl + "images/pushpins/plain-red.png";
+        
+        var position = new WorldWind.Position(55, -106, 1e2);
+        var placemark = new WorldWind.Placemark(position, false, placemarkAttributes);
+
+        placemark.label = "Placemark\n" +
```
Comment (eirizarry) [inline layouts/shortcodes/getStarted_FinalResult.html]: Would it be possible to spread the shapes/placemarks out more on the canvas? In a triangle shape. They look great, but a little squished together. I would love to see them taking up more space on the globe.

File: content/web/get-started.md [hunk 5]
```diff
@@ -7,44 +7,451 @@ draft: false
...
+wwd.addLayer(new WorldWind.CoordinatesDisplayLayer(wwd));
+wwd.addLayer(new WorldWind.ViewControlsLayer(wwd));
+{{< /highlight >}}
+
+Note that in order for both the corresponding icons of these controls to be displayed, as well as the low resolution Blue Marble imagery, we need their image files to be sourced locally. These need to be inside an '/images' directory which needs to be a sibling of the library, _i.e._ exist in the same directory. The image files can be downloaded in [here]({{% latestArtifactoryPath url="https://files.worldwind.arc.nasa.gov/artifactory" repo="web" %}}/images.zip).
 
 ---
 
-## NPM Package
+We have written a basic Web WorldWind app, but unlike the Java and Android versions of WorldWind, Web WorldWind is meant to be part of a web application, so we need a web server to visualize our work. If you're not familiar with local testing servers, [this is a good place to start](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server).
```
Comment (eirizarry) [inline content/web/get-started.md]: We may need to double check on a few citations just to make sure we are giving credit where we are supposed to.

File: content/web/get-started.md (hunk 3 above)
Comment (Beak-man) [reply content/web/get-started.md]: Yo mean the line streak? I thought appropriate to separate that tiny block of code from the rest. What do you think?

File: layouts/shortcodes/getStarted_FinalResult.html (hunk 4 above)
Comment (Beak-man) [reply layouts/shortcodes/getStarted_FinalResult.html]: Yes, definitely. I need to change all the shorctodes, will try it.

File: content/web/get-started.md (hunk 3 above)
Comment (eirizarry) [reply content/web/get-started.md]: I think we should delete the line streak if the text relates to what comes right before it, to keep continuity. Am I reading it correctly?

File: layouts/shortcodes/getStarted_FinalResult.html (hunk 4 above)
Comment (Beak-man) [reply layouts/shortcodes/getStarted_FinalResult.html]: Done, check if you like how it looks.

**Pull request merged**